docs/verifier/query-builder.md

```diff
@@ -2,59 +2,209 @@
 
 <div align="center">
 <img src={useBaseUrl("/img/query-builder.png")} align="center" width="600"/>
 </div>
-<br></br>
+<br/>
 
-The [Query Builder](https://schema-builder.polygonid.me/query-builder) contains the following fields:
+## Building a query
 
-- Proof type: credential issuance methods. SIG for Credentials with Baby JubJub(BJJ) Key Signatures and MTP for credentials with Merkle Tree Proof.
-- URL to JSON-LD Context: this should be the address where the JSON-LD Context is stored, containing schema data.
-- Schema Type: this is the field used to select the schema which the query is about, for credentials that have more than one schema type.
-- Issuer DID: refers to the identification of the issuer that generated the credential. Use `*` to consider any issuers. **We currently support only one value**, which means that you can either choose one particular Issuer DID or use `*` to accept any.
-- Attribute field: this is the attribute the query is based on. **Each query supports only one attribute**.
-- Query type: the two options of queries that are available, a Condition or Selective disclosure. Condition depends on a value and an operator; that would be the case of a query requesting a user to be older than 21 years old. [Selective Disclosure](/docs/verifier/features.md#selective-disclosure) is the feature used to ask the ID holder for a specific piece of their data.
-- Operator: this is the relation between the attribute and the value.
-- Value: this is the value the operator refers to.
-- Skip Revocation Check: will skip the check to see if a credential was revoked or not.
+### URL to JSON-LD Context
 
-You can see here how these fields should be filled for a verifier who needs to **check whether the credential holder is older than 21 years old**:
+To create a query, it is essential to reference the correct schema, specifically its JSON-LD Context's URL. Users have two options: they can either enter a custom URL directly or utilize the Schema Explorer. By beginning to type the name (or any other keyword) of the desired schema in the Schema Explorer, a list of available schemas will appear, allowing the user to select the appropriate one.
 
 <div align="center">
-<img src={useBaseUrl("/img/query-builder-filled.png")} align="center" width="600"/>
+<img src={useBaseUrl("/img/query-builder-url.png")} align="center" width="600"/>
 </div>
-<br></br>
+<br/>
 
-The [Query Builder](https://schema-builder.polygonid.me/query-builder) outcome should look like this:
+### Schema type
+
+The JSON-LD Context derived from a schema in the Schema Explorer is limited to a single type. However, a custom JSON-LD Context may encompass multiple types. This particular field enables users to select the appropriate type for constructing their query. As a user-friendly feature, in cases where the JSON-LD Context is single-typed, the field is pre-populated automatically.
```

Comment:
I would remove the `derived` keyword, the JSON-LD Context is not derived from anything...

> The JSON-LD Context from a schema in the Schema Explorer is limited to a single type.

Also, here `pre-populated` looks weird. When you fill a form you are filling the inputs, but not populating them:

> As a user-friendly feature, in cases where the JSON-LD Context is single-typed, the field is *pre-filled* automatically.

Reply:
I don't agree with these statements.

> I would remove the `derived` keyword, the JSON-LD Context is not derived from anything...

The JSON-LD Context is indeed derived from the schema in the Schema Explorer. The source of truth is the schema, the JSON LD Context is just a serialization of the schema. I agree though that this is an implementation detail that users don't care about.

> Also, here `pre-populated` looks weird. When you fill a form you are filling the inputs, but not populating them

[Populate](https://www.google.com/search?q=%22pre-populate+a+form%22) is perfectly correct in this context.

In any case, I added the two changes because I like you.